### ISSUE TITLE
Add support for alpha colour scale

### ISSUE THREAD

**Issue by gsouquet** [vector-im/compound#81]
We want to provide in parralel of the colour scale and alternative `alpha` one.

This is a good example that illustrates what we want to do: https://www.radix-ui.com/docs/colors/palette-composition/the-scales

- iOS: Potentially export a new color set, TBC by @pixlwave 
- Android: Using the HEXA format, so alpha can be handled easily
- Web: We can use the HEXA or `rgba` format

**Comment by pixlwave** [comment]
@gsouquet Yes please, if its simple enough to carry on as we are, then lets use new colour sets on iOS.

**Comment by janogarcia** [comment]
@gsouquet We'd need to generate the alpha colors in Colour Studio, so that we can easily export the tokens for Tokens Studio.

So far, I've used [Alphredo.app](https://alphredo.app/) to manually generate alpha version as needed.

Alphredo calculates the equivalent translucent color with the highest possible alpha value for against a specific background.

Keep in mind that not every foreground:background color combination can be converted to an alpha value, some physical limitations apply. For example, any color where any of the RGB channels is 0 can't be converted to an alpha value when against a white background (e.g., `#110000`,  `#111100`, `#FF0000` and so on against a white background).

Alphredo is not available as a npm package, but we could reuse a function from other libraries such as `transparentify()` from Stylus:

https://stylus-lang.com/docs/bifs.html#transparentify-top-bottom-alpha

**Comment by janogarcia** [comment]
On the token naming, from the designer experience side, it would be optimal following this structure for base tokens: 

- Solid: `color.gray.100`
- Alpha: `color.alpha.gray.100`

So, grouping all alpha colors in a parent `alpha` namespace, rather than appending it as a variant: `color.gray.100` and `color.gray.100.alpha`

If we append it, the style browser in Figma would be more overwhelming and confusing.
